Source/Lib/Codec/EbPictureDecisionProcess.c

```diff
@@ -1359,10 +1362,9 @@ void* PictureDecisionKernel(void *inputPtr)
                 // Remove the entry
                 if((inputEntryPtr->dependentCount == 0) &&
                    (inputEntryPtr->inputObjectPtr)) {
-                    EbReleaseObject(inputEntryPtr->pPcsPtr->pPcsWrapperPtr);
-                       // Release the nominal liveCount value
-                       EbReleaseObject(inputEntryPtr->inputObjectPtr);
```

Comment:
Shouldn't release the PPCS and its corresponding PA Ref objects here, because it may be still referenced by other pictures.